esmvaltool/preprocessor/__init__.py

```diff
@@ -22,7 +22,7 @@
                         fix_file, fix_metadata)
 from ._regrid import extract_levels, regrid
 from ._time import (extract_month, extract_season, seasonal_mean,
-                    time_average, extract_time)
+                    time_average, extract_time, annual_mean)
```

Comment:
sort imports alphabetically, you can do this by renaming the file, using isort and renaming it back

Reply:
I've fixed this by doing as you suggest by renaming the file. The good news is that isort is able to sort `__init__.py` files from version 4.3.5. (We're using 4.3.4 at the moment.)